controller/workitemtype_blackbox_test.go

```diff
@@ -93,6 +94,17 @@ var (
 	personID = uuid.FromStringOrNil("22a1e4f1-7e9d-4ce8-ac87-fe7c79356b16")
 )
 
+func getCategory() []*app.GenericData {
+	categoriesData := []*app.GenericData{}
+	catId := category.PlannerRequirementsID.String()
```

Comment:
Is it on purpose that you add only the requirements category and not the portfolio category? Also, this function needs a documentation.